### DISCUSSION TITLE
How to save tweet IDs from Stream into a list?

### DISCUSSION THREAD

**Discussion by mbjalaly** [tweepy/tweepy#1318] [Questions] [answered]
I implemented a code that streams live tweets using Stream. I would like to save the tweet IDs into a list that grows continuously. Appreciate your help :)

**Comment by Harmon758** [comment, marked as answer]
You can get the ID from a Status/[Tweet object](https://developer.twitter.com/en/docs/tweets/data-dictionary/overview/tweet-object) using its `id` attribute`.
